sui-execution/latest/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -1243,6 +1246,7 @@ mod checked {
     }
 
     pub fn finish(
+        vm: &MoveVM,
```

Comment:
We cannot support having a VM in this function--it is split out for the adapter rewrite

Reply:
ok - i was using it to go from `Type` to `TypeTag` - is there another way?